{{cookiecutter.project_shortname}}/run-tests.sh

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I (re)move this file to `scripts/test` just to follow completely https://github.com/github/scripts-to-rule-them-all, but this is just opinion and it can be change back to how it was, no hard feelings 😉